packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -872,7 +901,11 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 	public getPendingBlobs(): IPendingBlobs {
 		const blobs = {};
 		for (const [key, entry] of this.pendingBlobs) {
-			blobs[key] = { blob: bufferToString(entry.blob, "base64") };
+			blobs[key] = {
+				blob: bufferToString(entry.blob, "base64"),
+				uploadTime: entry.uploadTime,
+				minTTLInSeconds: entry.minTTLInSeconds,
```

Comment:
@anthony-murphy out of curiosity, do you know why typescript doesn't complain here? entry.minTTLInSeconds is optional, but minTTLInSeconds in IPendingBlobs is not.

Reply:
I actually missed the ? in IPendingBlobs.minTTLInSeconds, plus your comment about add them as non-optional members of an optional field makes sense to me. No idea why typescript didn't complain tho, but I'll change it anyway

Reply:
Actually there is not always the case where we have both values (minTTLInSeconds and uploadTime since only ODSP and local createBlobs have minTTLInSeconds. These values need to be independent for that reason.

Reply:
Do we need to track upload time if we don't get a TTL? I think no TTL implies blobs won't expire. Not something that needs to change in this PR, just wondering

Reply:
You're right. I made the change